source/Octopus.Server.Client/LocationChecker.cs

```diff
@@ -18,13 +16,14 @@ public static void CheckAssemblyLocation()
             {
                 if (octopusRegNode == null) 
                     return;
-                CheckBasedOnApplication(octopusRegNode, "Server");
-                CheckBasedOnApplication(octopusRegNode, "Tentacle");
+                // CheckBasedOnApplication(octopusRegNode, "Server");
```

Comment:
We should remove this commented out code
```suggestion
```

Reply:
Oh it is too! From the Octopus Server install directory
![image](https://user-images.githubusercontent.com/1687639/121439808-0175b480-c9ca-11eb-8a84-c0be0ac2c672.png)

Reply:
So we need to put this line back in (I was mistaken when I said we don't ship it with Server)